### PR TITLE
Remove Page Sidebar Field

### PR DESCRIPTION
**Pull request by mendelB**
### What's this PR do?

This pull request branches off of #2266 to remove references to the deprecated Page#sidebar field. It also adds the result of removing the field from Contentful Page and generating a new migration script. (Also removes two legacy Landing Page fields from the PHP Entity).

### How should this be reviewed?
👀 

### Any background context you want to provide?
We've deprecated any usage of this field in #2266 but broadly haven't been using this field for a hot minute. We'll also be removing the field from GraphQL in an accompanying PR.

### Relevant tickets

References [Pivotal #173738645](https://www.pivotaltracker.com/story/show/173738645). (Also deprecates two legacy Landing Page fields from the PHP Entity).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [x] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: app/Entities/LandingPage.php [hunk 1]
```diff
@@ -19,8 +19,6 @@ public function jsonSerialize()
             'fields' => [
                 'internalTitle' => $this->internalTitle,
                 'content' => $this->content,
-                'sidebar' => $this->parseBlocks($this->sidebar),
-                'blocks' => $this->parseBlocks($this->blocks),
```
Comment (katiecrane) [inline app/Entities/LandingPage.php:23]: Why is `blocks` removed here as well?

File: cypress/fixtures/contentful/exampleVoterRegistrationDriveCampaign.js [hunk 2]
```diff
@@ -122,8 +120,6 @@ export default {
       fields: {
         internalTitle: 'Online Registration Drive Landing Page',
         content: null,
-        sidebar: [],
-        blocks: [],
```
Comment (katiecrane) [inline cypress/fixtures/contentful/exampleVoterRegistrationDriveCampaign.js:126]: Same Q here about blocks being removed!

File: contentful/content-types/page.js [hunk 3]
```diff
@@ -267,22 +246,14 @@ module.exports = function(migration) {
   });
 
   page.changeFieldControl('metadata', 'builtin', 'entryLinkEditor', {});
-
-  page.changeFieldControl('authors', 'builtin', 'entryLinksEditor', {
-    bulkEditing: false,
-  });
+  page.changeFieldControl('authors', 'builtin', 'entryLinksEditor', {});
```
Comment (katiecrane) [inline contentful/content-types/page.js:249]: Is the update here an old one that got rolled in?

File: app/Entities/LandingPage.php (hunk 1 above)
Comment (mendelB) [reply app/Entities/LandingPage.php:23]: These fields were removed from the Landing Page content type in #1931 but I think we just forgot to remove them from the Entity here.

File: contentful/content-types/page.js (hunk 3 above)
Comment (mendelB) [reply contentful/content-types/page.js:249]: Yeah 🤷 I assume one of those formatting things that seem to get rolled in as we update these older scripts

**Pull request merged**